e2etest/connectivity_test.go

```diff
@@ -205,146 +202,45 @@ var _ = Describe("Connectivity", func() {
 			LabelSelector: metav1.FormatLabelSelector(helloDeployment.Spec.Selector),
 		})
 		Expect(err).ToNot(HaveOccurred())
-		var slicesv1 *discoveryv1.EndpointSliceList
-		var slicesv1b1 *discoveryv1beta1.EndpointSliceList
```

Comment:
These changes now also fulfill #51, so that PR can be closed if this happens to merge first.